examples/notebooks/2_Execution_Bell_circuit.ipynb

```diff
@@ -388,7 +390,7 @@
     }
    ],
    "source": [
-    "job_id, job = submit(circuit, IBMDevice.IBMQ_QASM_SIMULATOR)\n",
+    "job_id, job = submit(circuit, IBMDevice.IBM_LEAST_BUSY)\n",
```

Comment:
do we remove remote computer in the notebook ?

Reply:
I think we can let it, it appears clearly since we use `submit` no ?